aztk_cli/spark/endpoints/cluster/cluster_app_logs.py

```diff
@@ -23,5 +29,12 @@ def execute(args: typing.NamedTuple):
     if args.tail:
         utils.stream_logs(client=spark_client, cluster_id=args.cluster_id, application_name=args.app_name)
     else:
-        app_logs = spark_client.get_application_log(cluster_id=args.cluster_id, application_name=args.app_name)
-        print(app_logs.log)
+        app_log = spark_client.get_application_log(cluster_id=args.cluster_id, application_name=args.app_name)
+        if args.output:
+            spinner = utils.Spinner()
+            spinner.start()
+            with open(os.path.abspath(os.path.expanduser(args.output)), "w", encoding="UTF-8") as f:
+                f.write(app_log.log)
+            spinner.stop()
```

Comment:
Minor - probably want to wrap this in a try/catch block with the spinner.stop() in a finally statement in case writing to disk fails for any reason.

Reply:
Same comment in cluster submit below.